galasa-parent/dev.galasa.framework.api.common/src/main/java/dev/galasa/framework/api/common/ServletErrorMessage.java

```diff
@@ -30,9 +30,18 @@ public enum ServletErrorMessage {
 
     // GenericErrors...
     GAL5000_GENERIC_API_ERROR                       (5000,"E: Error occured when trying to access the endpoint. Report the problem to your Galasa Ecosystem owner."),
-    GAL5404_UNRESOLVED_ENDPOINT_ERROR               (5404,"E: Error occured when trying to identify the endpoint ''{0}''. Please check your endpoint URL or report the problem to your Galasa Ecosystem owner."),
     GAL5400_BAD_REQUEST                             (5400,"E: Error occured when trying to execute request ''{0}''. Please check your request parameters or report the problem to your Galasa Ecosystem owner."),
-    GAL5401_UNAUTHORIZED                            (5401,"E: Unauthorized. Please ensure you have provided a valid 'Authorization' header with a valid bearer token and try again.")
+    GAL5401_UNAUTHORIZED                            (5401,"E: Unauthorized. Please ensure you have provided a valid 'Authorization' header with a valid bearer token and try again."),
+    GAL5404_UNRESOLVED_ENDPOINT_ERROR               (5404,"E: Error occured when trying to identify the endpoint ''{0}''. Please check your endpoint URL or report the problem to your Galasa Ecosystem owner."),
+    GAL5405_METHOD_NOT_ALLOWED                      (5405,"E: Error occured when trying to access the endpoint ''{0}''. The method ''{1}'' is not allowed."),
+    GAL5411_NO_REQUEST_BODY                         (5411,"E: Error occured when trying to access the endpoint ''{0}''. The request body is empty."),
+
+    //CPS Namespaces
+    GAL5015_CPS_STORE_ERROR                         (5015,"E: Error occured when trying to access the Configuration Property Store. Report the problem to your Galasa Ecosystem owner."),
```

Comment:
This is an internal server error. Can we please add that at the front ?
eg: "Internal Server Error: ..." to make it clear it's not anything they have done, and it shouldn't ever occur.

Reply:
I'm hoping that the HTTP status code would be 500 for this.